README.md

```diff
@@ -31,56 +31,164 @@
 </div>
 <br />
 
-## A Go Implementation of the Polkadot Host
-
 > **Warning**
-> 2022-11-01: Gossamer is pre-production software
+>
+> The Gossamer Polkadot Host is pre-production software [2022-12-01]
 
-Gossamer is an implementation of the [Polkadot Host](https://wiki.polkadot.network/docs/learn-polkadot-host): an execution environment for the Polkadot runtime, which is materialized as a Web Assembly (Wasm) blob.  In addition to running an embedded Wasm executor, a Polkadot Host must orchestrate a number of interrelated services, such as [networking](dot/network/README.md), block production, block finalization, a JSON-RPC server, [and more](cmd/gossamer/README.md).
+Gossamer is a [Golang](https://go.dev/) implementation of the
+[Polkadot Host](https://wiki.polkadot.network/docs/learn-polkadot-host): an
+execution environment for the Polkadot runtime, which is materialized as a Web
+Assembly (Wasm) blob. In addition to running an embedded Wasm executor, a
+Polkadot Host must orchestrate a number of interrelated services, such as
+[networking](dot/network/README.md), block production, block finalization, a
+JSON-RPC server, [and more](cmd/gossamer/README.md#client-components).
 
-For more information about Gossamer, check out the [Gossamer Docs](https://ChainSafe.github.io/gossamer).
+## Getting Started
 
-## Get Started
+To get started with Gossamer, follow the steps below to build the source code
+and start a development network.
 
 ### Prerequisites
 
-Install Go version [`>=1.18`](https://go.dev/dl/#go1.18)
+[Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) is required
+to acquire the Gossamer source code, and
+[Make](https://tilburgsciencehub.com/building-blocks/configure-your-computer/automation-and-workflows/make/)
+is used to build it. Building Gossamer requires version 1.18 or greater of
+[Golang](https://go.dev/dl/).
 
 ### Installation
 
-get the [ChainSafe/gossamer](https://github.com/ChainSafe/gossamer) repository:
+Clone the [Gossamer](https://github.com/ChainSafe/gossamer) repository:
 
 ```
 git clone git@github.com:ChainSafe/gossamer
 cd gossamer
 ```
 
-build gossamer command:
+Build Gossamer:
 
 ```
 make gossamer
 ```
 
-### Troubleshooting for Apple Silicon users
+Or build Gossamer _and_ move the resulting executable to `$GOPATH/bin`:
+
+```
+make install
+```
+
+#### Troubleshooting for Apple Silicon users
 
-If you are facing the following problem with the `wasmer`:
+Apple Silicon users may encounter these errors:
 
 ```
 undefined: cWasmerImportObjectT
 undefined: cWasmerImportFuncT
 undefined: cWasmerValueTag
 ```
 
-Make sure you have the following Golang enviroment variables:
+If so, set the following
+[Golang environment variables](https://pkg.go.dev/cmd/go#hdr-Environment_variables):
+
+```
+GOARCH="amd64"
+CGO_ENABLED="1"
```

Comment:
This should be set for all platforms, not just apple silicon, so I'd remove from here?

```suggestion
```